src/protocolsupportpocketstuff/skin/SkinListener.java

```diff
@@ -28,13 +29,14 @@ public SkinListener(ProtocolSupportPocketStuff plugin) {
 		this.plugin = plugin;
 	}
 
-	@EventHandler
+	@EventHandler(priority = EventPriority.HIGHEST)
```

Comment:
Correct me if im wrong, but skinsrestorer will now really override this, monitor is ran last right...